src/test/java/ua/cv/tim/service/impl/AllianceServiceImplTest.java

```diff
@@ -0,0 +1,187 @@
+package ua.cv.tim.service.impl;
+
+
+
+import org.mockito.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import ua.cv.tim.dao.AllianceDao;
+import ua.cv.tim.dao.UserDao;
+import ua.cv.tim.dto.AllianceDTO;
+import ua.cv.tim.dto.UserDTO;
+import ua.cv.tim.model.Alliance;
+import ua.cv.tim.model.Player;
+import ua.cv.tim.model.Role;
+import ua.cv.tim.model.User;
+import ua.cv.tim.service.UserService;
+import ua.cv.tim.utils.SendMail;
+
+import javax.mail.MessagingException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+import static org.testng.Assert.*;
+
+/**
+ * Created by mmaksymtc on 26.01.2017.
+ */
+
+public class AllianceServiceImplTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(AllianceServiceImplTest.class);
+
+    @Mock
+    AllianceDao allianceDao;
+    @Mock
+    UserService userService;
+    @Mock
+    UserDao userDao;
+    @Mock
+    SendMail sendMail;
+    @InjectMocks
+    AllianceServiceImpl allianceService;
+    @Spy
+    List<Alliance> alliancesList = new ArrayList<>();
+    @Captor
+    ArgumentCaptor<Alliance> captor;
+    @BeforeClass
+    public void setUp(){
+        MockitoAnnotations.initMocks(this);
+        alliancesList= getAlliancesList();
+    }
+    @AfterMethod
+    public void resetAllMocks(){
+        Mockito.reset(allianceDao,userService,userDao,sendMail);
+    }
+    @Test
+    public void getAllTest(){
```

Comment:
test name made from real method name + suffix test, and method getAll from allianceServiceImpl class returns all alliances appropriately